docs/client-side/yeoman.rst

```diff
@@ -0,0 +1,188 @@
+
+1. Open Visual Studio 2015. From the File menu select :menuselection:`Open --> Project/Solution`.
+
+2. In the Open Project dialog, navigate to the ``project.json`` file, select it and click the **Open** button. In the Solution Explorer, the project should look something like the screenshot below.
+
+ .. image:: yeoman/_static/yeoman-solution.png
+ 
+.. note:: Yeoman creates a MVC web application complete with server and client side build support. Server-side dependencies are listed under the **References** node, and client-side dependencies in the **Dependencies** node/ of Solution Explorer. Dependencies are restored automatically when you load this project.
+
+ .. image:: yeoman/_static/yeoman-loading-dependencies.png 
+
+3.	When all the dependencies are restored, press **F5** to run the project. The default home page displays in the browser.
+ 
+ .. image:: yeoman/_static/yeoman-home-page.png 
+ 
+Restoring, Building and Hosting from the Command Line
+-----------------------------------------------------
+
+You can prepare and host your web application using commands **dnu** (Microsoft .NET Development Utility) and 
+**dnx** (Micorosft .NET Execution Environment). 
+
+.. note:: For more information on DNX see `DNX Overview <http://docs.asp.net/en/latest/dnx/overview.html>`_ 
+
+1. From the command line, change the current directory to the folder containing the project (that is, the folder that contains the `project.json` file). 
+
+.. code-block:: console
+
+ cd c:\MyYo\MyWebApp 
+ 
+2. From the command line, restore the project's package dependencies.   
+
+.. code-block:: console
+
+ dnu restore
+
+3. Also from the command line, build the project assemblies.   
+
+.. code-block:: console
+
+ dnu build
+
+4. To run the development web server run **dnx** command. 
+
+.. code-block:: console
+
+ dnx . web
+
+The web server will listen on port 5000. The URL and port are defined in project.json in the **commands** section.
+
+.. code-block:: json
+	:emphasize-lines: 2
+
+	"commands": {
+		"web": "Microsoft.AspNet.Hosting --server Microsoft.AspNet.Server.WebListener --server.urls http://localhost:5000",
+		"kestrel": "Microsoft.AspNet.Hosting --server Kestrel --server.urls http://localhost:5001",
+		"gen": "Microsoft.Framework.CodeGeneration",
+		"ef": "EntityFramework.Commands"
+	},
+
+5. Open a web browser and navigate to http://localhost:5000. 
+
+ .. image:: yeoman/_static/yeoman-home-page_5000.png 
+
+.. note:: You can also run the cross-platform Kestrel development server using the **dnx . kestrel** command. By default, Kestrel listens on port 5001 as defined in the **project.json**, **commands** section.  
+
+Adding to Your Project with Sub Generators
+------------------------------------------
+You can add new generated files using Yeoman even after the project is created. Use `sub generators <https://www.npmjs.com/package/generator-aspnet#sub-generatorsl>`_ to add any of the file types
```

Comment:
Bad link - should be https://www.npmjs.com/package/generator-aspnet#sub-generators